src/graphql/thegraph/utils.ts

```diff
@@ -0,0 +1,77 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
+import ms from 'ms'
+
+/**
+ * Used to get large amounts of data when
```

Comment:
it looks like ian might've lost his trail of thought here, could we finish the comment for him :dead: